web/src/components/onboarding/taxId/DisabledTaxId.tsx

```diff
@@ -110,8 +110,16 @@ export const DisabledTaxId = (props: Props): ReactElement => {
 
   const getSpacedValue = (value: string): ReactNode => (
     <>
-      <span className={taxIdDisplayStatus === "password-view" ? "text-ls-3" : ""}>{value.slice(0, 9)}</span>
-      <span>{value.slice(9)}</span>
+      {taxIdDisplayStatus === "password-view" ? (
+        <span data-testid={"disabled-tax-id-value"} className={"password-view"}>
+          {"****-****-****"}
+        </span>
+      ) : (
+        <>
+          <span data-testid={"disabled-tax-id-value"}>{value.slice(0, 9)}</span>
+          <span>{value.slice(9)}</span>
```

Comment:
If we are just displaying it in plain text, I think we can change it just being in just one span right? we don't need to slice it anymore?

Reply:
correct and done